org.eclipse.scanning.points/src/org/eclipse/scanning/points/MultiStepGenerator.java

```diff
@@ -14,6 +14,7 @@
 import java.text.MessageFormat;
 
 import org.eclipse.scanning.api.ModelValidationException;
+import org.eclipse.scanning.api.annotation.ui.FieldDescriptor;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.eclipse.scanning.api.annotation.ui.FieldDescriptor'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)